### PR TITLE
JBQA-6163 Link Workspace Selection to Forge Runtime implementation

### PR DESCRIPTION
**Pull request by jkopriva**
Changes:
- Added better output in EvaluateBrowserPage method
- Added condition ForgeConsoleHasNoChange
- Added condition ProjectIsSelected
- Added tests for Link Workspace Selection to Forge Runtime

### CODE REVIEW

File: plugins/org.jboss.tools.forge.reddeer/src/org/jboss/tools/forge/reddeer/condition/ForgeConsoleHasNoChange.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+/**
+ * Returns true if a console has no change for the specified time period.
+ * 
+ * @author jkopriva@redhat.com
+ * 
+ */
+public class ForgeConsoleHasNoChange extends AbstractWaitCondition {
+
+	private TimePeriod timePeriod;
```
Comment (rawagner) [inline plugins/org.jboss.tools.forge.reddeer/src/org/jboss/tools/forge/reddeer/condition/ForgeConsoleHasNoChange.java]: is this timePeriod used anywhere ?

File: plugins/org.jboss.tools.forge.reddeer/src/org/jboss/tools/forge/reddeer/condition/ProjectIsSelected.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+	/*
+	 * (non-Javadoc)
+	 * 
+	 * @see org.jboss.reddeer.common.condition.WaitCondition#test()
+	 */
+	@Override
+	public boolean test() {
+		ProjectExplorer pe = new ProjectExplorer();
+		pe.open();
```
Comment (rawagner) [inline plugins/org.jboss.tools.forge.reddeer/src/org/jboss/tools/forge/reddeer/condition/ProjectIsSelected.java]: you should open ProjectExplorer in constructor. Code will be more performant

**Comment by jkopriva** [comment]
@rawagner thx for review, I have updated PR

File: plugins/org.jboss.tools.forge.reddeer/src/org/jboss/tools/forge/reddeer/condition/ProjectIsSelected.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+
+	/*
+	 * (non-Javadoc)
+	 * 
+	 * @see org.jboss.reddeer.common.condition.WaitCondition#test()
+	 */
+	@Override
+	public boolean test() {
+		pe.open();
```
Comment (rawagner) [inline plugins/org.jboss.tools.forge.reddeer/src/org/jboss/tools/forge/reddeer/condition/ProjectIsSelected.java]: can you please move this line too ? :)

File: tests/org.jboss.tools.forge.ui.bot.test/src/org/jboss/tools/forge2/ui/bot/wizard/test/DeployScaffoldDBTest.java [hunk 4]
```diff
@@ -48,6 +48,8 @@
 	private List<String> tableNames = new ArrayList<String>();
 
 	private static final String PACKAGE = GROUPID + ".model";
+	
+	private static final String SERVER_NAME = "WildFly 10.x Server";
```
Comment (rawagner) [inline tests/org.jboss.tools.forge.ui.bot.test/src/org/jboss/tools/forge2/ui/bot/wizard/test/DeployScaffoldDBTest.java]: And one last thing I just noticed - you should not assume that RedDeer will always create a server with this name. Correct way of obtaining sever name is as follows:

Inject requirement:
`@InjectRequirement
    private ServerRequirement sr;
`

and later in your code:
`sr.getServerNameLabelText(sr.getConfig())`

Comment (jkopriva) [inline tests/org.jboss.tools.forge.ui.bot.test/src/org/jboss/tools/forge2/ui/bot/wizard/test/DeployScaffoldDBTest.java]: thx, I have added this to my code:)

**Pull request merged**